packages/cmf/src/UIRouter.js

```diff
@@ -25,23 +24,20 @@ import api from './api';
  * @return {object} ReactElement
  */
 const UIRouter = (props, context) => {
-	const routes = api.route.getRoutesFromSettings(context, props.routes);
-	if (routes.path === '/' && !!routes.component) {
-		return (<BaseRouter routes={routes} history={props.history} />);
+	const routes = api.route.getRoutesFromSettings(context, props.routes, props.dispatch);
```

Comment:
The UIRouter is just connected, not cmfConnected so you don't have props.dispatch.
You may have it from the context  if you add the store to it

Reply:
Yes but it's ok IMO. The entry point to cmf store is still redux, so dispatching an action from react-redux is the same as dispatching an action from cmf redux.